index.html

```diff
@@ -5,6 +5,16 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link rel="stylesheet" href="main.css">
     <title>Battle Game Project</title>
+<<<<<<< HEAD
+  </head>
+  <body>
+    <div class="start-game">
+      <input class="player-name-input" type="text" name="player-name">
+      <button class="begin-journey-button"type="button" name="start-game-button">Begin Journey</button>
+      <div class="background-img-homepage">
+        <img src="images/richard-tilney-bassett-183124.jpg">
```

Comment:
I think background images should go in the CSS since it's just styling and not functional for the site.